app/assets/stylesheets/earthworks.css

```diff
@@ -134,26 +134,48 @@
   outline: none;
 }
 
+.search-q .input-group {
+  padding: .5rem
+}
+
 .input-group > .search-autocomplete-wrapper {
   border: none;
 }
 
-.search-query-form, .navbar-search .search-query-form{
+.input-group > .search-autocomplete-wrapper input {
+  padding-top: 0;
+  padding-bottom: 0
```

Comment:
```suggestion
  padding-block: 0;
```

Reply:
I don't want all 4 sides to be zero. I don't want to override the left and right padding.